docs/contents/hooks/use-format-byte.ja.mdx

```diff
@@ -0,0 +1,89 @@
+---
+title: useFormatByte
+description: "`useFormatByte`はByteを人間が読めるようにフォーマットするためのカスタムフックです。"
+package_name: "@yamada-ui/use-format-byte"
+with_description: true
+label: New
+---
+
+## インポート
+
+```ts
+import { useFormatByte } from "@yamada-ui/react"
+```
+
+## 使い方
+
+:::note
+`useFormatByte`は、内部的に[Intl.NumberFormat](https://developer.mozilla.org/ja/docs/Web/JavaScript/Reference/Global_Objects/Intl/NumberFormat)を使用しています。
+:::
+
+### 基本
+
+```tsx functional=true
+const formattedBytes = useFormatByte(1024, "en-US")
+
+return <Text>{formattedBytes}</Text>
+```
+
+### サイズ
+
+```tsx functional=true
+const formatted50Bytes = useFormatByte(50, "en-US")
+const formatted5KB = useFormatByte(5000, "en-US")
+const formatted5MB = useFormatByte(5000000, "en-US")
+const formatted5GB = useFormatByte(5000000000, "en-US")
+
+return (
+  <>
+    <Text>{formatted50Bytes}</Text>
+    <Text>{formatted5KB}</Text>
+    <Text>{formatted5MB}</Text>
+    <Text>{formatted5GB}</Text>
+  </>
+)
+```
+
+### Bitsからフォーマット
```

Comment:
It might make sense to use katakana here too, since everything else is in katakana.
```suggestion
### ビットからフォーマット
```

Reply:
Fixed.